nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/main/java/org/nuxeo/ecm/platform/rendition/service/RenditionServiceImpl.java

```diff
@@ -497,4 +519,55 @@ private void removeStoredRenditions(Map<String, List<String>> liveDocumentRefsTo
         }
     }
 
+    @Override
+    public Rendition getDefaultRendition(DocumentModel doc, String reason, Map<String, Serializable> extendedInfos) {
+        Rendition rendition = null;
+        if (defaultRenditionDescriptor != null) {
```

Comment:
You should do early-exit `if (defaultRenditionDescriptor == null) { return null; }` to avoid having a big indented block.